book/core-concepts/partitioning.md

```diff
@@ -117,14 +117,14 @@ The partition function takes in message data and returns a partition key. In the
 def partition(data):
     return data.symbol
 {%- language name="Go", type="go" -%}
-func symbolToKey(symbol string) uint64 {
-    return uint64(binary.BigEndian.Uint32([]byte(fmt.Sprintf("%4s", symbol))))
+func symbolToKey(symbol string) byte {
```

Comment:
This code isn't quite right. It's returning a `byte` instead of a `[]byte`. Take a look at the code in `testing/performance/apps/go/market_spread/go/src/market_spread/market_spread/symbol_partition.go` to see what we do now. You'll also need to fix up the `Partition` function below (but all you need to do there is change `byte` to `[]byte`).